README.md

```diff
@@ -1,13 +1,23 @@
-# Gifted Chat
+<p align="center">
+    <img alt="react-native-linkedin" src="https://raw.githubusercontent.com/FaridSafi/react-native-gifted-chat/master/screenshots/gifted-chat-1.png" width=200>
+    <img alt="react-native-linkedin" src="https://raw.githubusercontent.com/FaridSafi/react-native-gifted-chat/master/screenshots/gifted-chat-2.png" width=200>
+</p>
+
+<h3 align="center">
+  💬 <strong>Gifted Chat</strong> <br/>
+  The most complete chat UI for React Native <br/>
+  <small>(formerly known as Gifted Messenger).</small>
+</h3>
+<p align="center">
+  <a href="https://www.npmjs.com/package/react-native-gifted-chat">
+  <img alt="npm dowloads" src="https://img.shields.io/npm/dm/react-native-gifted-chat.svg"/></a>
```

Comment:
These images appear to be broken

Reply:
This is a copy past image from before page! 
If you find better image feel free to change!

Reply:
I see, the site must have been down! They seem to be loading for me now.